lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1680,6 +1726,9 @@ namespace Js
 
         ScriptContext* scriptContext = arrayPrototype->GetScriptContext();
         JavascriptLibrary* library = arrayPrototype->GetLibrary();
+
+        library->EnsureIndexOfByteCode();
```

Comment:
It will be one of the part of my mission if we want to use this builtin system. For the moment Michael and Louis told me to don't "really" care about it.

---
In reply to: [111827172](https://github.com/Microsoft/ChakraCore/pull/2845#discussion_r111827172) [](ancestors = 111827172)